deploy/olm-catalog/redhat-marketplace-operator/0.2.0/marketplace.redhat.com_meterbases_crd.yaml

```diff
@@ -99,6 +99,11 @@ spec:
         status:
           description: MeterBaseStatus defines the observed state of MeterBase.
           properties:
+            availableReplicas:
```

Comment:
Remove this file from the PR, it'll change a released version in our catalog. Not quite sure how this change got in there.